lib/matplotlib/style/__init__.py

```diff
@@ -1,3 +1,6 @@
 from __future__ import absolute_import
 
 from .core import use, context, available, library, reload_library
+from matplotlib import rcParams
+if rcParams['style']:
+    use(rcParams['style'])
```

Comment:
I don't think this is the appropriate place for this call. IIUC, the default style isn't used unless the `style` module is imported.